README.adoc

```diff
@@ -201,18 +206,18 @@ First, create a test class in the `src/test/java/it/io/openliberty/guides/metric
 include::finish/src/test/java/it/io/openliberty/guides/metrics/MetricsTest.java[tags=MetricsTest]
 ----
 
-* The `testGetPropertiesTime()` test case validates the `@Timed` metric. It sends a request to the
+* The `testPropertiesRequestTimeMetric()` test case validates the `@Timed` metric. It sends a request to the
 `http://localhost:9080/inventory/systems/localhost` URL to access the `inventory` service, which adds
 the `localhost` host to the inventory. Next, the test case makes a connection to the
 `https://localhost:9443/metrics/application` URL to retrieve application metrics as plain text.
-It then uses the `validateMetric()` method to assert whether the time that is needed to retrieve
+Then it asserts whether the time that is needed to retrieve
```

Comment:
Introductory words like "However" and "Then" should be followed by a comma when they begin a sentence.
"Then, it asserts..."

Reply:
updated